clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonClusterProvider.groovy

```diff
@@ -86,15 +86,12 @@ class AmazonClusterProvider implements ClusterProvider<AmazonCluster> {
     String imageId = launchConfigs?.attributes?.get('imageId')
     CacheData imageConfigs = imageId ? cacheView.get(IMAGES.ns, Keys.getImageKey(imageId, account, region)) : null
 
-    def serverGroup = new AmazonServerGroup(name: name, region: region)
+    def serverGroup = new AmazonServerGroup(serverGroupData.attributes)
     serverGroup.accountName = account
```

Comment:
The result from `/clusters/{cluster}/serverGroups/{serverGroup}` does not contain `accountName` ... but that should be the only difference.

Previously we had a number of null attributes being returned from `/serverGroups/{serverGroup}` that were there when fetched via the cluster.